examples/tutorial/exercises/Exercise-3-networks-and-geoviews.ipynb

```diff
@@ -284,7 +283,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "graph_opts = opts.Graph(width=800 height=800, edge_selection_line_color='black', edge_hover_line_color='red',\n",
+    "graph_opts = opts.Graph(width=800, height=800, edge_selection_line_color='black', edge_hover_line_color='red',\n",
```

Comment:
Presumably this wasn't even runnable?  Scary that this got through our checks, but I guess we don't build the exercises. :-(

Reply:
yeah. Not great. I think nbsmoke should catch this.

Reply:
^ Obviously. I mean that maybe we should be sure to set nbsmoke to fail rather than warn.

Reply:
Yes, seems like there should be a category of issues that are clear failures, regardless of whether other issues are mere warnings.